tests/lib/nested.sh

```diff
@@ -1153,13 +1153,19 @@ nested_start_core_vm_unit() {
     PARAM_RTC="${NESTED_PARAM_RTC:-}"
     PARAM_EXTRA="${NESTED_PARAM_EXTRA:-}"
 
+    local EXPECT_PASSPHRASE
+    EXPECT_PASSPHRASE=${NESTED_EXPECT_PASSPHRASE:-}
+
     # Open port 7777 on the host so that failures in the nested VM (e.g. to
     # create users) can be debugged interactively via
     # "telnet localhost 7777". Also keeps the logs
     #
     # XXX: should serial just be logged to stdout so that we just need
     #      to "journalctl -u $NESTED_VM" to see what is going on ?
     if "$QEMU" -version | grep '2\.5'; then
+        if [ -z "$EXPECT_PASSPHRASE" ]; then
+            echo "internal error: NESTED_EXPECT_PASSPHRASE is set and qemu doesn't support chardev over socket"
```

Comment:
do you remember which systems this old version appears on?

Reply:
Apparently it is xenial https://launchpad.net/ubuntu/xenial/+source/qemu.